test/end-to-end/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "end-to-end",
+  "version": "0.0.0",
+  "description": "weldr-web end-to-end automation test",
+  "scripts": {
+    "test": "mocha -R xunit -O output=result.xml"
+  },
+  "dependencies": {
```

Comment:
OK, but why do you need to repeat the list of end-to-end dependencies in the top level package.json file?